tide_search.install

```diff
@@ -141,3 +141,14 @@ function tide_search_update_8004() {
   );
   $tideSearchOperation->addSearchableFieldsTerms();
 }
+
+/**
+ * Update search api fields for Ripple 2.0 search.
+ */
+function tide_search_update_8005() {
```

Comment:
this update needs to be added in the install config as well for the bran new cms first install.

Reply:
thanks, latest update now has this on the default yml.